config.w32

```diff
@@ -224,6 +246,34 @@ if (PHP_MONGODB != "no") {
     WARNING("mongodb libsasl support not enabled, libs not found");
   }
 
+  if (PHP_CLIENT_SIDE_ENCRYPTION != "no") {
+    mongoc_opts.MONGOC_ENABLE_CLIENT_SIDE_ENCRYPTION = 1;
+
+    ADD_SOURCES(configure_module_dirname + "/src/libmongocrypt/src", PHP_MONGODB_MONGOCRYPT_SOURCES, "mongodb");
+    ADD_SOURCES(configure_module_dirname + "/src/libmongocrypt/kms-message/src", PHP_MONGODB_MONGOCRYPT_KMS_MESSAGE_SOURCES, "mongodb");
+    ADD_SOURCES(configure_module_dirname + "/src/libmongocrypt/src/os_win", PHP_MONGODB_MONGOCRYPT_WIN_SOURCES, "mongodb");
+
+    if (mongoc_ssl_found) {
+      mongoc_opts.MONGOCRYPT_ENABLE_CRYPTO = 1;
+      mongoc_opts.MONGOCRYPT_ENABLE_CRYPTO_OPENSSL = 1;
+
+      ADD_SOURCES(configure_module_dirname + "/src/libmongocrypt/src", "crypto/openssl.c", "mongodb");
+      ADD_SOURCES(configure_module_dirname + "/src/libmongocrypt/kms-message/src", "kms_crypto_openssl.c", "mongodb");
```

Comment:
I have some questions about needing to manually include these source files separately and whether or not they're in the original list produced by `find`. Asked that in a separate thread on `CheckSSL.m4`.